tracker/tracker/src/main/app/observer/observer.ts

```diff
@@ -206,6 +207,7 @@ export default abstract class Observer {
   }
 
   private unbindTree(node: Node) {
+    let removed = 0
```

Comment:
Plz, move just before `while` so the variable usage remains compact

Reply:
done